bftengine/src/bftengine/PersistentStorageImp.cpp

```diff
@@ -905,18 +897,20 @@ void PersistentStorageImp::verifyLastNewViewMsgs(const DescriptorOfLastNewView &
   }
 }
 
-void PersistentStorageImp::verifySetDescriptorOfLastNewView(const DescriptorOfLastNewView &desc) const {
```

Comment:
Why are this and other verify methods no longer const ?

Reply:
hasDescriptorOfLastExecution changes data member.